lib/packetgen/config.rb

```diff
@@ -20,12 +21,12 @@ class Config
     # @return [String] 
     attr_reader :ip6addr
 
-    # Create a configuration object. If +iface+ is not set, get first network interface.
-    # If non exists, use loopback one.
+    # Create a configuration object. If +iface+ is not set then 
+    # it will be attempted to be found automatically. 
     # @param [String,nil] iface
     def initialize(iface=nil)
       if iface.nil?
-        iface = NetworkInterface.interfaces.select { |iface| iface != 'lo' }.first
+        iface = Pcap.lookupdev
         iface = NetworkInterface.interfaces.first if iface.nil?
```

Comment:
CI tests are broken because `Pcap.lookupdev` does not fall back on lo interface, and in CI environment, there is only a loopback network interface.
Solution seems to fall back on loopack interface if `Pcap.lookupdev` returns nothing:
```ruby
begin
  iface = Pcap.lookupdev
rescue PCAPRUB::BindingError
  iface = NetworkInterface.interfaces.select { |iface| iface =~ /lo/ }
end
```
Could you update you PR, please?